### ISSUE TITLE
configure site with GA

### ISSUE THREAD

**Issue by thekaveman** [cal-itp/reports#9]
The GA property is already set up, just need to add some `<script>` to the `<head>` section:

```html
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-Q4W9YVZW3C"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-Q4W9YVZW3C');
</script>
```

**Comment by hunterowens** [comment]
added in #14

**Comment by thekaveman** [comment]
Closed by #14
